projects/BFDR/IJENatality.cs

```diff
@@ -4937,12 +4956,43 @@ public string ATTEND_NPI
         {
             get
             {
-                // TODO: Implement mapping from FHIR record location: 
-                return "";
+                string fhirFieldName = "NPI";
```

Comment:
can you call the LeftJustified_Get here instead? I noticed you specify the length as 10 but it looks like the IJE field info says its 12. Does it say it should be 10 somewhere?

Reply:
I can change it, the length of 10 was an oversight on my part